README.md

```diff
@@ -112,7 +112,46 @@ would throw a stylelint error similar to `All rules have already been disabled (
 
 #### Interpolation linting
 
-We do not currently support linting interpolations as it could be a big performance hit though we aspire to have at least partial support in the future. You can of course lint your own mixins in their separate files, but it won't be linted in context, the implementation currently just inserts relevant dummy values. This, we are afraid, means you won't be able to lint cases such as `declaration-block-no-duplicate-properties` etc. and won't be able to lint outside mixins such as [polished](https://github.com/styled-components/polished).
+We are afraid that we don't support linting interpolations (meaning the actual CSS that would be inserted into an interpolation through a variable or function during runtime) as it could be a big performance hit. You can of course lint your own mixins in their separate files, but it won't be linted in context, our implementation simply inserts relevant dummy values. This, we are afraid, means you won't be able to lint cases such as `declaration-block-no-duplicate-properties` etc. and won't be able to lint outside mixins such as [polished](https://github.com/styled-components/polished) in context.
```

Comment:
I would get rid of this paragraph since it's unnecessary, the following paragraphs explain how it works anyway.